web/channels/notification_channel.ex

```diff
@@ -0,0 +1,9 @@
+defmodule Habits.NotificationChannel do
+  @moduledoc false
+
+  use Habits.Web, :channel
+
+  def join("notifications", _, socket) do
+    {:ok, socket}
+  end
+end
```

Comment:
Not much going on here, but I should still learn about [`Phoenix.ChannelTest`](https://hexdocs.pm/phoenix/Phoenix.ChannelTest.html)